src/ripple/rpc/handlers/ServerInfo.cpp

```diff
@@ -38,6 +40,17 @@ doServerInfo(RPC::JsonContext& context)
         context.params.isMember(jss::counters) &&
             context.params[jss::counters].asBool());
 
+    if (context.app.config().reporting())
```

Comment:
I think this can be simplified into a try ... catch such as:
try:
    ret[field] = proxied[result][jssinfo][load_factor]
catch:
  ret[field] = 1